launcher/FileSystem.cpp

```diff
@@ -350,12 +351,12 @@ QString getDesktopDir()
 }
 
 // Cross-platform Shortcut creation
-bool createShortCut(QString location, QString dest, QStringList args, QString name, QString icon)
+bool createShortcut(QString destination, QString target, QStringList args, QString name, QString icon)
 {
-#if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
-    location = PathCombine(location, name + ".desktop");
+#if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD) || defined(Q_OS_OPENBSD)
+    destination = PathCombine(destination, name + ".desktop");
```

Comment:
```suggestion
    destination += ".desktop";
```

Reply:
`destination` is the folder the shortcut will end up in. This will cause the `.desktop` file to be `/home/user/Desktop.desktop` (or wherever Qt thinks the desktop is).

Reply:
nvm, disregard my earlier comment